assets/src/edit-story/components/postLock/postLock.js

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { useFeatures } from 'flagged';
+import { trackError } from '@web-stories-wp/tracking';
+
+/**
+ * Internal dependencies
+ */
+import { useAPI } from '../../app/api';
+import { useStory } from '../../app/story';
+import { useConfig } from '../../app/config';
+import { useCurrentUser } from '../../app/currentUser';
+import PostLockDialog from './postLockDialog';
+import PostTakeOverDialog from './postTakeOverDialog';
+
+function PostLock() {
+  const {
+    actions: { getStoryLockById, setStoryLockById, deleteStoryLockById },
+  } = useAPI();
+  const {
+    state: { currentUser },
+  } = useCurrentUser();
+  const {
+    storyId,
+    dashboardLink,
+    nonce: firstNonce,
+    postLock: { interval: postLockInterval, showLockedDialog },
```

Comment:
@spacedmonkey This needs to be addded to `DEFAULT_CONFIG` in the karma fixture to make sure the tests pass.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/6497/commits/935bb08aba3e4ea35818db4a2f0b5dce1de2532a